framework/components/ema-list-items/ema-list-items.twig

```diff
@@ -1 +1,41 @@
 {% include '@ec-europa/ecl-list-items' %}
+
+{% if variant == 'translation' %}
+  <section class="{{ _css_class }}"{{ _extra_attributes|raw }}>
```

Comment:
Also we should not add classes and extra attributes here (we only add them to the component itself, i.e list-item).
But it could be useful to add a specific class for translations section (something like `ema-list-item__translations`)